packages/api/local/server.ts

```diff
@@ -10,7 +10,7 @@ const apiOptions = {
   platform: 'vtex',
   account: 'storeframework',
   environment: 'vtexcommercestable',
-  channel: '1',
+  channel: '{"salesChannel":"1"}',
```

Comment:
why a string and not an object?

Reply:
Because as a string all platform handle variables that vary the channel inside this channel string

Reply:
but channel is something explicitly of the vtex platform, and the way vtex uses it, there are several variables, maybe the best way is to call `channel` from `channels` and it be an object that receives sales = 1.

Reply:
I think it's not, because the types will vary according to the platform if the value is an object. We've discussed it before, the way we wanna handle channels. @tlgimenes suggested this way.

Reply:
@emersonlaurentino does it make sense?